frontend/src/components/CountdownTimer.module.scss

```diff
@@ -0,0 +1,40 @@
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+@import "../styles/tokens.scss";
+
+.countdown-timer {
+  dl {
+    display: flex;
+    gap: $spacing-sm;
+
+    > div {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      gap: $spacing-xs;
+      width: 60px;
```

Comment:
Could we use `$layout-lg` instead? It's `64px`.

Reply:
Oh that's right, I'd forgotten about that one! 1410c3aa